imio/dms/mail/browser/settings.py

```
@@ -1027,6 +1035,63 @@ def validate_settings(data):  # noqa
+                    if sub[date_fld]:
+                        try:
+                            parsed[date_fld] = datetime.datetime.strptime(sub[date_fld], "%Y/%m/%d")
+                        except ValueError:
+                            raise Invalid(
+                                _(
+                                    u"${tab} tab: « ${field} », substitute ${rule} has an invalid date "
+                                    u"in « ${date} » field",
+                                    mapping={"tab": _(u"Outgoing mail"), "field": _(u"Signer substitutes"),
+                                             "rule": i, "date": _(date_title)},
+                                )
+                            )
```

Ruff (0.15.9)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)